ui/app/AppLayouts/ActivityCenter/ActivityCenterLayout.qml

```diff
@@ -0,0 +1,774 @@
+import QtQuick
+import QtQuick.Layouts
+
+import StatusQ.Components
+import StatusQ.Controls
+import StatusQ.Core
+import StatusQ.Core.Backpressure
+import StatusQ.Core.Theme
+import StatusQ.Popups.Dialog
+import StatusQ.Core.Utils as StatusQUtils
+import StatusQ.Layout
+
+import shared
+import shared.popups
+import shared.views.chat
+
+import utils
+
+import AppLayouts.Chat.stores as ChatStores
+import AppLayouts.Profile.stores
+import AppLayouts.stores.Messaging as MessagingStores
+import AppLayouts.stores.Messaging.Community as CommunityStores
+import AppLayouts.stores
+import AppLayouts.ActivityCenter.helpers
+
+import "views"
+import "panels"
+
+StatusSectionLayout {
+    id: root
+
+    property ContactsStore contactsStore
+    property ActivityCenterStore activityCenterStore
+    property ChatStores.RootStore store
+    property PrivacyStore privacyStore
+    property NotificationsStore notificationsStore
+    property MessagingStores.MessagingRootStore messagingRootStore
+
+    QtObject {
+        id: d
+
+        readonly property var loadMoreNotificationsIfScrollBelowThreshold: Backpressure.oneInTimeQueued(root, 100, function() {
+            if (listView.contentY >= listView.contentHeight - listView.height - 1) {
+                root.activityCenterStore.fetchActivityCenterNotifications()
+            }
+        })
+
+        readonly property bool isStatusNewsViaRSSEnabled: root.privacyStore.isStatusNewsViaRSSEnabled
+        readonly property var notificationsSettings: root.notificationsStore.notificationsSettings
+
+        // TODO: Review if it's really necessary to do this call later call or instead directly mark the notification as seen
+        // Now temporarily kept as it was done when closing the previous popup
+        function callLaterMarkAsSeen() {
+            Qt.callLater(activityCenterStore.markAsSeenActivityCenterNotifications)
+        }
+    }
+
+    leftPanel: ColumnLayout {
+
+        objectName: "activityCenterLeftPanel"
+
+        id: leftPanel
+        anchors.fill: parent
+        spacing: 0
+
+        // TODO: Create Header Component
+        RowLayout {
+            id: row
+
+            Layout.fillWidth: true
+            Layout.leftMargin: Theme.bigPadding
+            Layout.topMargin: Theme.padding
+
+            StatusNavigationPanelHeadline {
+                text: qsTr("Notifications")
+            }
+
+            // Filler
+            Item {
+                Layout.fillWidth: true
+            }
+
+            StatusBetaTag {
+                tooltipText: qsTr("Under construction.<br>More notification types to be coming soon.")
```

Comment:
Do we still want to display this @jrainville ? It looks silly, it's been like that for years...

Reply:
Maybe now that we are touching it and it's a wip.. can make sense to keep it for some time more?!